src/Resources/views/tailwind_2/_table.html.twig

```diff
@@ -173,29 +183,35 @@
                             data-entity-id="{{ row.id }}">
                     </td>
                 {% endif %}
+                {% if subTables %}
+                    <td>
+                        <span
+                            class="block px-6 py-2 cursor-pointer"
+                            {{ stimulus_action('araise/table-bundle/accordion', 'toggle', 'click') }}
+                            {{ stimulus_target('araise/table-bundle/accordion', 'arrow') }}
+                        >
+                            <svg
+                                class="inline-block align-baseline h-2.5 w-2.5 text-neutral-600 mr-2"
+                                xmlns="http://www.w3.org/2000/svg"
+                                viewBox="0 0 8.836 14.036"
+                            >
+                                <path id="arrow-right" d="M27.65,1.333a.718.718,0,0,1,0-1.1.97.97,0,0,1,1.251,0l7.067,6.238a.717.717,0,0,1,0,1.1L28.9,13.808a.97.97,0,0,1-1.251,0,.718.718,0,0,1,0-1.1l6.487-5.685Z" transform="translate(-27.391)" fill="#4b5563" fill-rule="evenodd"/>
```

Comment:
Should the color (fill) in the `path` be `fill="currentColor"`? Seems odd it has already a static colour assigned to then being overwritten directly by a class.